lang/nl/texts/data.en.html

```diff
@@ -26,59 +26,59 @@ <h3>MongoDB dump</h3>
 <dl>
  <dt>Link</dt>
  <dd><a href="https://static.openfoodfacts.org/data/openfoodfacts-mongodbdump.tar.gz">https://static.openfoodfacts.org/data/openfoodfacts-mongodbdump.tar.gz</a></dd>
- <dt>sha256sum</dt>
+ <dt>sha256som</dt>
```

Comment:
This probably should not be translated unless we phrase the source differently.
```suggestion
 <dt>sha256sum</dt>
```